test/videojs-contrib-hls.test.js

```diff
@@ -2780,7 +2780,7 @@ QUnit.test('cleans up the buffer when loading VOD segments', function(assert) {
   this.clock.tick(1);
   this.player.currentTime(120);
   this.player.tech_.hls.mediaSource.sourceBuffers[0].trigger('updateend');
-  this.clock.tick(1);
+  this.clock.tick(2);
```

Comment:
This change was needed because previously, after an `updateend` event, `SegmentLoader.handleUpdateEnd_` would call `SegmentLoader.monitorBuffer_` which would setup a timeout to call `SegmentLoader.fillBuffer_` on the next clock tick, but with the addition of calling `load` on a seek, this would clear that timeout and set a new one for the next clock tick.